amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/settings/SettingsConstants.java

```diff
@@ -49,6 +49,8 @@
     public static final String CUSTOM = "custom";
 
     public static final String REORDER_FUNDING_ITEM_ID = "reorder-funding-item";
+    public static final String WORKSPACE_CURRENCY_CODE = "workspace-" + CURRENCY_ID;
```

Comment:
From what I see, this will be effective currency code, either for the one customized for the current ws or the global one including for public without ws. So may be to simply use `CURRENCY_ID` without this extra constant? Or at least a more general one like `effective-currency-code`